examples/dummy/asyncapi.yaml

```
@@ -1,25 +1,32 @@
-asyncapi: 2.0.0
+asyncapi: 3.0.0
 info:
   title: Account Service
   version: 1.0.0
   description: This service is in charge of processing user signups
 servers:
   mosquitto:
-    url: mqtts://test.mosquitto.org
+    host: test.mosquitto.org
     protocol: mqtts
     security:
-      - X509Certificate: []
+      - $ref: '#/components/securitySchemes/X509Certificate'
     bindings:
       mqtt:
         clientId: guest
         cleanSession: true
         keepAlive: 60
-        bindingVersion: 0.1.0
+        bindingVersion: 0.2.0
   websockets:
-    url: ws://localhost:3005/ws
+    host: 'localhost:3005'
+    pathname: /ws
     protocol: ws
 channels:
   user/signedup:
```

Remove `/` to avoid confusions, please.

```suggestion
  userSignedUp:
```